cluster/src/main/scala/io/snappydata/gemxd/SparkSQLExecuteImpl.scala

```diff
@@ -483,7 +483,7 @@ object SnappyContextPerConnection {
     val context = connectionIdMap.get(connectionID)
     if (context != null) context
     else {
-      val snc = SnappyContext.getOrCreate(null: SparkContext).newSession()
+      val snc = SnappyContext(null: SparkContext).newSession()
```

Comment:
>>I don't think separate JDBC connections should be sharing the same UI tab 

It would mean that for every client connection we will be creating a new tab. Also, there would be a different tab for every job? Why would we want to do that when Storage and Job tabs are already shared. 
IMO, this is a hack but from usability point of view it is needed. My vote would be to keep the newSession as is for now and make the necessary changes only when we merge with 2.0. 

>>We can add something to UI to group multiple connections in a single UI tab to reduce number of tabs if number of connections is large. Can be tracked separately for 0.6/1.0 

UI Listener has been moved to SparkSession in 2.0. So 2.0, I believe, we will have a clean solution. 
I would recommend deferring this PR.